### PR TITLE
test: ✅ xunit init and tests for webapi

### LINKED ISSUE #89: Setup xUnit for the WebApi project
Setup an xUnit project to test the `WebApi`.

### PR DESCRIPTION
**Pull request by tfisicaro**
closes #89

### CODE REVIEW

File: src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Xunit;
+
+namespace Congo.WebApi.Tests
+{
+    public class CartControllerTests
+    {
+        private readonly Mock<IMediator> mediatorMoq;
```
Comment (pobiega) [inline src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs]: This should follow the same naming pattern as the rest of the project.
```suggestion
        private readonly Mock<IMediator> _mediatorMoq;
```

File: src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        private CartController _subject;
```
Comment (pobiega) [inline src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs]: In your other tests, this is readonly. It should be that here too :) 
```suggestion
        private readonly CartController _subject;
```

**Review by pobiega — CHANGES_REQUESTED** [review]
Minor changes needed, but over all good.

File: src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs (hunk 1 above)
Comment (tfisicaro) [reply src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs]: Refactored all tests to use this pattern.

File: src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs (hunk 2 above)
Comment (tfisicaro) [reply src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs]: Fixed

**Review by pobiega — APPROVED** [review]

**Pull request merged**